docs/whats_new.rst

```diff
@@ -34,6 +35,10 @@ Changelog
 ~~~~~~~~~
 - Nothing yet
 
+Bug
+~~~
+- Fixed RANSAC to give consistent results with a fixed seed across different chunk sizes, by `Austin Hurst`_ and `Yorguin Mantilla`_ (`#43 <https://github.com/sappelhoff/pyprep/pull/43>`_)
```

Comment:
actually, can you please add an additional (new) "API" section underneath the "bug" section where you mention that the parameters have changed for RANSAC methods? i.e., `n_pred_chns` has been dropped from methods X, Y, Z, ... It should refer to the same PR.

```RST
API
~~~
- did that and that .., by `Austin Hurst`_ and `Yorguin Mantilla`_ (`#43 <https://github.com/sappelhoff/pyprep/pull/43>`_)


```